makefile

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/make
```

Comment:
I went through all that work in GCC4MBED to get rid of recursive makefiles and now we go and add one back? If we really want the makefile to be in the root directory then it is probably better to just move the one currently in src/ and update a few things to let it know that the src can be found in src/ instead of ./